helper/useragent/useragent.go

```diff
@@ -32,3 +32,45 @@ func String() string {
 	return fmt.Sprintf("Vault/%s (+%s; %s)",
 		versionFunc(), projectURL, rt)
 }
+
+// AgentString returns the consistent user-agent string for Vault Agent.
+//
```

Comment:
can we remove this empty line or is this intentional?

Reply:
This was how the formatting for the Vault, so I kept things consistent:
```
// String returns the consistent user-agent string for Vault.
//
// e.g. Vault/0.10.4 (+https://www.vaultproject.io/; go1.10.1)
```